docs/guide/query_scan.md

```diff
@@ -514,19 +514,7 @@ worker can be a thread (in programming languages that support multithreading) or
 system process. To perform a parallel scan, each worker issues its own Scan request with an
 unique `WorkerId`.
 
-=== "Kotlin - SDK 2.x"
-    
-    ```kotlin
-    Not supported
-    ```
-
-=== "Java - SDK 2.x"
-
-    ```java
-    Not supported
-    ```
-
-=== "Kotlin - SDK 1.x"
+=== "Kotlin"
```

Comment:
I think you still need to include examples for SDK2 and SDK1 variants, even though the APIs for both is pretty much the same.

I'm just pattern matching what the documentation is doing.

Reply:
Good catch! This is true.

Reply:
Hm, I don't think that's true. This was the only section that had the v1/v2 breakdown